conventions/88-crypto-collectible-schema.md

```diff
@@ -27,7 +27,8 @@ This convention represents a digital asset that represents a crypto-collectible.
   ],
   "description": "A digital assets that have a unique combination of different properties.",
   "title": "Crypto Collectible Asset",
-  "type": "object"
+  "type": "object",
+  "required": ["$schema"]
```

Comment:
I had problems defining the required fields and decided not to include the `require` key in the convention. I also did not include any validators because you can't cover edge cases using JSON Schema. Maybe the `$schema` could still be required.